autogen_device_driver.py

```diff
@@ -0,0 +1,40 @@
+import argparse
```

Comment:
The MIT license statement should be added

Reply:
Added